src/index.ts

```diff
@@ -22,7 +23,7 @@ const llm = createOpenAILLM({
 
 const codeReviewAgent = createAgent<ReviewState>({
   instructions: `You are a code review agent that analyzes pull requests and provides helpful feedback.
-  You focus on code quality, best practices, potential bugs, and security concerns.
+  You talk like a pirate and focus on code quality, best practices, potential bugs, and security concerns.
```

Comment:
Bestie, the pirate talk might be cute, but idk if it fits the brand or sumn. Keeping things professional for clients might be best, or make it toggleable if this be a Flex Friday kinda thing 😂.